modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractJavaCodegen.java

```diff
@@ -550,6 +550,7 @@ public void processOpts() {
             setDateLibrary(additionalProperties.get("dateLibrary").toString());
         }
 
+        additionalProperties.put("dateLibrary_" + dateLibrary, "true");
```

Comment:
In the new code, if you look at line 555, 556, 562, 568, etc, we already add mustache tags for date libraries. why do we need `dateLibrary_${dateLibrary}` in partciuarl?